fe/fe-core/src/main/java/com/starrocks/planner/HdfsScanNode.java

```diff
@@ -430,4 +430,8 @@ protected void toThrift(TPlanNode msg) {
         }
     }
 
+    @Override
+    public boolean canUsePipeLine() {
```

Comment:
Nit: /s/canUsePipeLine/canUsePipeline

Reply:
Yes. Should be `Pipeline`. But I don't want to make too much change on this PR:)